resources/ts/Pages/Integrations/Detail.tsx

```diff
@@ -34,9 +37,9 @@ const Detail = ({ ...integration }: Props) => {
   }, []);
 
   const integrationTypesInfo = useIntegrationTypesInfo();
-  const integrationTypeInfo = integrationTypesInfo.find(
+  const { Icon } = integrationTypesInfo.find(
     (i) => i.type === integration.type
-  );
+  )!;
```

Comment:
What is the exclamation mark doing a the end?

Reply:
@simon-debruijn I added a plain map from type to icon which is easier to statically analyse for TS than an array lookup so it removes the need to bend the types

Reply:
Great 👍🏻